flask/app.py

```diff
@@ -421,7 +421,7 @@ def __init__(self, import_name, static_path=None, static_url_path=None,
         #: A dictionary with lists of functions that should be called after
         #: each request.  The key of the dictionary is the name of the blueprint
         #: this function is active for, ``None`` for all requests.  This can for
-        #: example be used to open database connections or getting hold of the
+        #: example be used to close database connections or getting hold of the
         #: currently logged in user.  To register a function here, use the
         #: :meth:`after_request` decorator.
```

Comment:
The mention of current user doesn't really make sense here either.  Maybe just change it to "close database connections and other open resources"?  Or just remove the second part entirely.